### PR TITLE
Upgrade to 1.5.2

### PR DESCRIPTION
**Pull request by saraedum**
including a patch from upstream that is already in Sage

### CODE REVIEW

**Comment by saraedum** [comment]
@conda-forge-admin, please rerender.

**Comment by saraedum** [comment]
~~This depends on https://github.com/conda-forge/fflas-ffpack-feedstock/pull/5.~~ (merged)

**Comment by saraedum** [comment]
@isuruf: This looks good to me. What do you think?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,10 +8,12 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://github.com/linbox-team/linbox/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 47f025fe7aab204fb276307f32a562de357c887c6ef589b9e4f7a57a3b58ecaf
+  sha256: 7b69ab617b15ca6684b29d08531ed8899cd4d07ec75799e5d624c4c0a60dc6b2
+  patches:
+    - patches/9e6e5dd-f4b483c.patch
```
Comment (isuruf) [inline recipe/meta.yaml]: Can you add a comment here about where this patch is coming from?

Comment (saraedum) [reply recipe/meta.yaml]: Something like this?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -8,10 +8,13 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://github.com/linbox-team/linbox/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 47f025fe7aab204fb276307f32a562de357c887c6ef589b9e4f7a57a3b58ecaf
+  sha256: 7b69ab617b15ca6684b29d08531ed8899cd4d07ec75799e5d624c4c0a60dc6b2
+  patches:
+    # An upstream patch that Sage uses for 8.2
+    - patches/9e6e5dd-f4b483c.patch
```
Comment (isuruf) [inline recipe/meta.yaml]: Do you have a link for the patch?

Comment (saraedum) [reply recipe/meta.yaml]: Ok.

Comment (isuruf) [reply recipe/meta.yaml]: So it's not from upstream? Was it sent upstream by sage developers?

Comment (saraedum) [reply recipe/meta.yaml]: It's originally from Sage but it is already in upstream master in the exact same form.

**Pull request merged**